.github/workflows/test-sample-apps.yml

```diff
@@ -42,10 +46,13 @@ jobs:
       - name: Build Java apps
         run: ./http2/gradlew clean build -p http2
       - name: Logging into target CloudFoundry
+        if: (github.event_name == 'pull_request' && github.actor != 'dependabot[bot]') || (github.event_name == 'pull_request_target' && github.actor == 'dependabot[bot]')
```

Comment:
not sure if these are still needed, if yo declare the same condition already on the overall job?

If they're not necessary, please remove. If they are, please ignore this comment.

Reply:
decided that it's needed the other way around. When a job condition does not match, it is skipped and marked as "Successful". We don't want that.